Mathlib/Data/Rel.lean

```diff
@@ -262,8 +355,42 @@ namespace Function
 def graph (f : α → β) : Rel α β := fun x y => f x = y
 #align function.graph Function.graph
 
+theorem graph_id : graph id = @Eq α := by simp[graph]
+
+theorem graph_comp {f : β → γ} {g : α → β} : graph (f ∘ g) = Rel.comp (graph g) (graph f) := by
+  ext x y
+  simp[graph, Rel.comp]
+
 end Function
 
+theorem Equiv.graph_inv (f : α ≃ β) : f.invFun.graph = Rel.inv (f.toFun.graph) := by
```

Comment:
For complicated typeclass reasons, we use `f.symm` instead of `f.invFun` for Equivs, and cast them instead of writing `toFun`:
```suggestion
theorem Equiv.graph_inv (f : α ≃ β) : (f.symm : β → α).graph = Rel.inv (f : α → β).graph := by
```